pkg/ui/react-app/src/thanos/Navbar.tsx

```diff
@@ -40,6 +40,8 @@ const navConfig: { [component: string]: (NavConfig | NavDropDown)[] } = {
     { name: 'Alerts', uri: '/new/alerts' },
     { name: 'Rules', uri: '/new/rules' },
   ],
+  bucket: [{ name: 'Blocks', uri: '/new/blocks' }],
+  compact: [{ name: 'Blocks', uri: '/new/blocks' }],
```

Comment:
This is copy-paste right?

Reply:
Yes. Like for now, bucket web and compactor has same menus.